src/server/req_runjob.c

```diff
@@ -985,11 +985,15 @@ svr_strtjob2(job *pjob, struct batch_request *preq)
 
 	/* send the job to MOM */
 
+	/* clearing old comment since we are sending a fresh job */
+	job_attr_def[(int) JOB_ATR_Comment].at_decode(
+			&pjob->ji_wattr[(int) JOB_ATR_Comment], (char *) 0,
+			(char *) 0, "");
```

Comment:
I agree. This is a change in behavior. However, I feel we should not set the comment as "Job run" there, since the job has not actually run yet, and the comment could immediately get replaced by a mom reject message. I like the "job about to run" comment. That is more consistent with the state. But the point is, we need to clear (by setting the above new message) the rejected message from the older cycle, since that could confuse somebody seeing a R/41 with "mom rejected" message.